src/Orleans.Runtime/MembershipService/DevelopmentMembershipOptions.cs

```diff
@@ -0,0 +1,13 @@
+using System.Net;
+
+namespace Orleans.Hosting
+{
+    /// <summary>Configures development clustering options</summary>
+    public class DevelopmentMembershipOptions
+    {
+        /// <summary>
+        /// Gets or sets the seed node to find the membership system grain.
```

Comment:
"seed node" is still in xml comments. I think it should be primary silo node?

Reply:
I wanted to keep this explicitly as a different way of referring to the same thing... otherwise if I just say primary again, it doesn't add anything extra

Reply:
that's fair